doc/testing.rst

```diff
@@ -0,0 +1,79 @@
+Automating the tests for your App
+=================================
+
+This an introductory guide to writing automated tests for your
+Morepath app.  We assume you've already installed Morepath; if not,
+see the :doc:`installation` section.
+
+In order to carry out the test we'll use WebTest_, which you'll need
+to have installed:
+
+.. code-block:: sh
+
+  $ pip install webtest
+
+
+Testing a basic app
+-------------------
+
+Let’s look at a minimal test of a basic app in Morepath:
+
+.. testcode::
+
+  def test_basic():
+      from morepath.tests.fixtures import basic
+      from webtest import TestApp as Client
+
+      c = Client(basic.app())
+
+      response = c.get('/foo')
+
+      assert response.body == b'The view for model: foo'
+
+You can save this function into a file and use a test automation tool
+like pytest_ to run it, or you can also invoke it as a regular Python
```

Comment:
Show how someone would invoke pytest. I think we should just assume pytest is installed and works for this.